sgv2-service-common/src/main/java/io/stargate/sgv2/common/grpc/DefaultStargateBridgeClient.java

```diff
@@ -90,15 +96,16 @@ public Response executeBatch(Batch request) {
   }
 
   @Override
-  public CqlKeyspaceDescribe getKeyspace(String keyspaceName) throws UnauthorizedKeyspaceException {
-    if (!authorizeSchemaRead(SchemaReads.keyspace(keyspaceName, SchemaRead.SourceApi.REST))) {
+  public Optional<CqlKeyspaceDescribe> getKeyspace(String keyspaceName)
+      throws UnauthorizedKeyspaceException {
```

Comment:
Switched to optional. As much as I'm comfortable using null in the cache, this is a higher level API exposed to other modules.

Also, I made `UnauthorizedKeyspaceException` (and its new table counterpart) unchecked. There is a catch-all mapper `BridgeAuthorizationExceptionMapper` in the web layer. This make it easier to deal with those errors, in particular in code that composes optionals.